src/main/kotlin/racingcar/model/Game.kt

```diff
@@ -0,0 +1,49 @@
+package racingcar.model
+
+import racingcar.ui.OutputView
+import kotlin.random.Random
+
+class Game(numberOfCar: Int) {
+    private val listOfCar: ArrayList<Car> = arrayListOf()
+    private val outputView: OutputView = OutputView()
+
+    init {
+        for (times in 1..numberOfCar) {
+            listOfCar.add(Car())
+        }
```

Comment:
repeat을 쓰는게 조금 더 가독성이 좋을 것 같아요.
```suggestion
        repeat(numberOfCar) {
            listOfCar.add(Car())
        }
```